spec/system/admin/shared_context.rb

```diff
@@ -1,9 +1,10 @@
 RSpec.shared_context "admin", shared_context: :metadata do
   before(:all) do
-    if ENV["STAGING"] 
+    if (ENV["STAGING"] = 1)
```

Comment:
Comparison would need to be `==` - you're assigning inline here (surprised rubocop doesn't yell?)

However I'd leave this as it was, and remove `STAGING: 0` above... - `0` is still _truthy_ in ruby, but also I think these values come in as strings regardless, so it's `"0"` (truthy in most languages I think [except PHP])